yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/ProfileResponse.java

```diff
@@ -2,19 +2,23 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-class ProfileResponse {
+public class ProfileResponse {
 
-    @JsonProperty("session_data")
+    @JsonProperty(Property.SESSION_DATA)
     private String sessionData;
 
-    @JsonProperty("receipt")
+    @JsonProperty(Property.RECEIPT)
     private Receipt receipt;
 
-    public ProfileResponse(){}
+    @JsonProperty(Property.ERROR_DETAILS)
+    private ErrorDetails error;
 
-    private ProfileResponse(String sessionData, Receipt receipt){
+    public ProfileResponse() { }
+
+    private ProfileResponse(String sessionData, Receipt receipt, ErrorDetails error) {
         this.sessionData = sessionData;
         this.receipt = receipt;
+        this.error = error;
     }
```

Comment:
1. Why have the empty `public` ctor?
2. Why not pass the builder instance to the private ctor?